account-kit/core/src/store/types.ts

```diff
@@ -56,7 +53,26 @@ export type SignerStatus = {
   isDisconnected: boolean;
 };
 
-export type ClientState = {
+export type StoredState =
+  | Omit<StoreState, "signer" | "accounts">
```

Comment:
I think this was here for backwards compatibility. going forward we should keep the stored state as the type below this one